phpstan.neon

```diff
@@ -512,10 +513,209 @@ parameters:
+                - packages/node-type-resolver/src/FileSystem/CurrentFileInfoProvider.php # 22
+                - packages/node-type-resolver/src/FileSystem/CurrentFileInfoProvider.php # 35
+                - packages/phpstan-static-type-mapper/src/TypeMapper/ObjectTypeMapper.php # 123
+                - packages/phpstan-static-type-mapper/src/TypeMapper/ObjectWithoutClassTypeMapper.php # 72
+                - packages/rector-generator/src/Rector/Closure/AddNewServiceToSymfonyPhpConfigRector.php # 39
+                - rules/nette-code-quality/src/FormControlTypeResolver/AssignedVariablesMethodCallsFormTypeResolver.php # 50
+                - rules/nette-code-quality/src/FormControlTypeResolver/ClassMethodFormTypeResolver.php # 67
+                - rules/nette-code-quality/src/FormControlTypeResolver/ConstructorFormControlTypeResolver.php # 62
+                - rules/nette-code-quality/src/FormControlTypeResolver/GetComponentMethodCallFormControlTypeResolver.php # 110
+                - rules/nette-code-quality/src/FormControlTypeResolver/MagicNetteFactoryInterfaceFormControlTypeResolver.php # 95
+                - rules/nette-code-quality/src/FormControlTypeResolver/MethodCallFormControlTypeResolver.php # 59
+                - rules/nette-code-quality/src/FormControlTypeResolver/NewFormControlTypeResolver.php # 61
+                - rules/nette-code-quality/src/FormControlTypeResolver/ReturnFormControlTypeResolver.php # 54
+                - rules/nette-code-quality/src/FormControlTypeResolver/ThisVariableInAnotherMethodFormControlTypeResolver.php # 52
+                - rules/nette-code-quality/src/FormControlTypeResolver/VariableConstructorFormControlTypeResolver.php # 84
+                - src/Configuration/ChangeConfiguration.php # 17
+                - src/Configuration/Configuration.php # 149
+                - src/Configuration/Configuration.php # 172
+                - src/Configuration/Configuration.php # 204
+                - src/Configuration/Configuration.php # 227
+                - src/Configuration/CurrentNodeProvider.php # 16
+                - src/HttpKernel/RectorKernel.php # 70
+                - src/Rector/AbstractRector/CallableNodeTraverserTrait.php # 24
+                - src/Rector/AbstractRector/CallableNodeTraverserTrait.php # 24
+                - src/Rector/AbstractRector/ConstFetchAnalyzerTrait.php # 24
+                - src/Rector/AbstractRector/ConstFetchAnalyzerTrait.php # 24
+                - src/Testing/NodeVisitor/AttributeCollectingNodeVisitor.php # 24
+
+        -
+            message: '#Do not use factory in constructor#'
+            paths:
+                - src/Testing/PHPUnit/Runnable/ClassLikeNamesSuffixer.php # 27
+                - src/Testing/PHPUnit/Runnable/RunnableClassFinder.php # 29
+                - utils/node-documentation-generator/src/ValueObject/NodeInfo.php # 29
+
+        -
+            message: '#Method with @required need to be named autowire\+class name#'
```

Comment:
@samsonasik This seems like a lot of work. We'll need a Rector rule for this.
Could you handle it?

Reply:
I will try